disnake/interactions/application_command.py

```diff
@@ -371,7 +371,7 @@ class ApplicationCommandInteractionDataResolved:
         IDs and messages
     """
 
-    __slots__ = ("members", "users", "roles", "channels", "messages")
+    __slots__ = ("members", "users", "roles", "channels", "messages", "attachments")
```

Comment:
The documentation for `attachment` is missing above.

Reply:
Fixed in f77f5a7de6d24433eac90792bb634367367e12c3, thanks